src/server/controllers/ApiController.js

```diff
@@ -24,12 +25,33 @@ export default class ApiController {
         : response(res, 'Login failed', 404))
       .catch(err => response(res, err.message, 401));
   };
+
+  /**
+   *
+   * @param {string} refresh
+   * @param {string} userId
+   */
+  static refreshTokenHandler (req, res) {
+    const { refresh, userId } = req.body;
+
+    const findedRefreshToken = availableTokens
+      .find((token, index) => token.ref === refresh);
+
+    if (findedRefreshToken) {
+      let index = availableTokens.indexOf(findedRefreshToken);
```

Comment:
You can use const here, if you don’t reassign this value

Reply:
done